bigquery/vacancy-feedback-metrics-by-month.sql

```diff
@@ -1,83 +1,115 @@
-  CAST(SUM(hires_rate_through_tv_upperbound*vacancies_published) AS INT64) AS hires_through_tv_upperbound,
-  CAST(SUM(exclusive_hires_rate*vacancies_published) AS INT64) AS exclusive_hires,
-  CAST(SUM(exclusive_hires_rate_upperbound*vacancies_published) AS INT64) AS exclusive_hires_upperbound,
-  CAST(SUM(exclusivity_rate*vacancies_published) AS INT64) AS exclusive_listings,
-  SUM(exclusive_hires_rate*vacancies_published*IF(category IN ("teacher","leadership"),1200,0)) AS savings, #assume that only teachers and leadership exclusive hires save schools £1200/vacancy, and that other vacancies do not
-  
-  SAFE_DIVIDE(SUM(exclusive_hires),SUM(feedback_available))*SUM(vacancies_published)*1200 AS raw_savings, #estimate of savings, assuming all categories of vacancy behave the same and save schools the same amount of money
-
-  # Extrapolations from hiring staff feedback to estimates of total numbers of vacancies which were hires, exclusive etc, unnormalised (i.e. assuming all categories of vacancy behave the same)
-  SUM(hires_through_tv) AS hires_through_tv_feedback,
-  SUM(hires_through_tv_upperbound) AS hires_through_tv_upperbound_feedback,
-  SUM(exclusive_hires) AS exclusive_hires_feedback,
-  SUM(exclusive_hires_upperbound) AS exclusive_hires_upperbound_feedback,
-  SUM(exclusive_listings) AS exclusive_listings_feedback,
-
-  #Extrapolations from hiring staff feedback to estimates of the overall proportion of vacancies which were hires, exclusive etc, normalised by category
-  SUM(hires_rate_through_tv*vacancies_published)/SUM(vacancies_published) AS hires_rate_through_tv,
-  SUM(hires_rate_through_tv_upperbound*vacancies_published)/SUM(vacancies_published) AS hires_rate_through_tv_upperbound,
-  SUM(exclusive_hires_rate*vacancies_published)/SUM(vacancies_published) AS exclusive_hires_rate,
-  SUM(exclusive_hires_rate_upperbound*vacancies_published)/SUM(vacancies_published) AS exclusive_hires_rate_upperbound,
-  SUM(exclusivity_rate*vacancies_published)/SUM(vacancies_published) AS exclusivity_rate,
-
-  #Extrapolations from hiring staff feedback to estimates of the overall proportion of vacancies which were hires, exclusive etc, unnormalised (i.e. assuming all categories of vacancy behave the same)
-  SAFE_DIVIDE(SUM(hires_through_tv),SUM(feedback_available)) AS raw_hires_rate_through_tv,
-  SAFE_DIVIDE(SUM(hires_through_tv_upperbound),SUM(feedback_available)) AS raw_hires_rate_through_tv_upperbound,
-  SAFE_DIVIDE(SUM(exclusive_hires),SUM(feedback_available)) AS raw_exclusive_hires_rate,
-  SAFE_DIVIDE(SUM(exclusive_hires_upperbound),SUM(feedback_available)) AS raw_exclusive_hires_rate_upperbound,
-  SAFE_DIVIDE(SUM(exclusive_listings),SUM(feedback_available)) AS raw_exclusivity_rate,
-
+  *,
+  #calculate the confidence intervals for these key rates, assuming a 95% confidence level (i.e. z=1.96), that they can be modelled as binomial distributions and that the normal distribution approximation can be applied.
```

Comment:
It would be easier to review this if the comment was wrapped onto multiple lines. In GitHub I can't read the comment on one line:
<img width="1349" alt="Screenshot 2020-09-09 at 14 42 23" src="https://user-images.githubusercontent.com/60350599/92606277-ad0e9e80-f2aa-11ea-9333-60190434eb45.png">

Reply:
Actually, there are a lot of too-long-to-read lines in this file, so I will just check out the branch and read it in my editor, as I don't know how easy it is to limit the length of SQL lines.

Reply:
@david-mears-dfe I'll split the comments out, but the lines aren't as easy to split - because BigQuery handles the spacing and splitting itself as part of its autoformat function.